bigjobcreator.py

```diff
@@ -1,7 +1,5 @@
-import os
```

Comment:
@thiagomanel do you know why this script is here? **_bigjobcreator_** looks like a load test right?

Reply:
@hukasu @IgorVictor do you know why this script is here? bigjobcreator looks like a load test right?